src/Morphir/JsonSchema/Backend.elm

```diff
@@ -105,55 +105,144 @@ extractTypes modName definition =
             )
 
 
-mapTypeDefinition : TypeName -> Type.Definition ta -> Maybe ( TypeName, SchemaType )
-mapTypeDefinition typeName definition =
+mapTypeDefinition : ( Path, Name ) -> Type.Definition ta -> List ( TypeName, SchemaType )
+mapTypeDefinition (( path, name ) as qualifiedName) definition =
     case definition of
-        Type.TypeAliasDefinition typeArgs typ ->
-            mapType typ
-                |> Maybe.map (Tuple.pair typeName)
-
-        Type.CustomTypeDefinition typeArgs accessControlledConstructors ->
-            Nothing
-
-
-mapType : Type ta -> Maybe SchemaType
+        Type.TypeAliasDefinition _ typ ->
+            [ ( mapQualifiedName qualifiedName
+              , case mapType typ of
+                    Ok val ->
+                        val
+
+                    _ ->
+                        Null
```

Comment:
You are ignoring the errors here and returning `Null` which is something we wanted to avoid.

Reply:
Fixed now